codespell_lib/data/dictionary.txt

```diff
@@ -31653,6 +31654,7 @@ lastest->latest, last,
 lastr->last
 lates->later, latest,
 latets->latest
+latice->lattice
```

Comment:
```suggestion
latice->lattice
latices->lattices
```

Reply:
My wrong, [unlike `latice`](http://app.aspell.net/lookup?dict=en_US-large;words=latice), a variant of `lattice` or `lettuce` [according to OED](https://www.oed.com/search/dictionary/?q=latice), plural [`latices` can be found in SCOWL and (Friends)](http://app.aspell.net/lookup?dict=en_US-large;words=latices), probably as the [plural of `latex`](https://www.oed.com/search/dictionary/?q=latices).